kubelet/tests/test_cadvisor.py

```diff
@@ -21,21 +21,18 @@
 ]
 
 
-@pytest.fixture
-def aggregator():
-    from datadog_checks.stubs import aggregator
-    aggregator.reset()
-    return aggregator
+@pytest.fixture()
```

Comment:
why did you change this?

Reply:
I meant to put this in the PR description, basically there is a bug in requests-mock where the decorator isn't registered as a fixture for pytest, and this only impacts PY3 - https://github.com/pytest-dev/pytest/issues/2749